JWTDecode.xcodeproj/project.pbxproj

```diff
@@ -146,6 +146,7 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				D41DECE32DC89DE100F5B1A4 /* CwlPreconditionTesting in Frameworks */,
```

Comment:
Please remove the CwlPreconditionTesting dependency as well (IIRC, it was added as an SPM dependency).